test/tests/functions/string.json

```
@@ -44,6 +44,16 @@
           "type": "unresolved-variable"
         }
       ]
+    },
+    {
+      "description": "NFC: keys are normalized",
+      "src": ".local $x = {\u1E0A\u0323 :string} .match $x \u1E0A\u0323 {{Not normalized}} \u1E0C\u0307 {{Normalized}} * {{Wrong}}",
+      "expErrors": [{"type": "duplicate-variant"}]
+    },
```

We should also validate that quoted keys are compared to unquoted keys according to their resolved value.

```suggestion
    {
      "description": "NFC: keys are normalized (unquoted)",
      "src": ".local $x = {\u1E0A\u0323 :string} .match $x \u1E0A\u0323 {{Not normalized}} \u1E0C\u0307 {{Normalized}} * {{Wrong}}",
      "expErrors": [{"type": "duplicate-variant"}]
    },
    {
      "description": "NFC: keys are normalized (quoted)",
      "src": ".local $x = {\u1E0A\u0323 :string} .match $x |\u1E0A\u0323| {{Not normalized}} |\u1E0C\u0307| {{Normalized}} * {{Wrong}}",
      "expErrors": [{"type": "duplicate-variant"}]
    },
    {
      "description": "NFC: keys are normalized (mixed)",
      "src": ".local $x = {\u1E0A\u0323 :string} .match $x \u1E0A\u0323 {{Not normalized}} |\u1E0C\u0307| {{Normalized}} * {{Wrong}}",
      "expErrors": [{"type": "duplicate-variant"}]
    },
```